capstone/src/test/java/com/google/sps/data/DataStoreTest.java

```diff
@@ -0,0 +1,48 @@
+
+import static com.google.appengine.api.datastore.FetchOptions.Builder.withLimit;
+import static org.junit.Assert.assertEquals;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class DataStoreTest {
+
+  private final LocalServiceTestHelper helper =
+      new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+
+  @Before
+  public void setUp() {
+    helper.setUp();
+  }
+
+  @After
+  public void tearDown() {
+    helper.tearDown();
+  }
+
+  // Run this test twice to prove we're not leaking any state across tests.
+  private void doTest() {
```

Comment:
Might want to rename this to some more specific to what you are testing than `doTest()`. For example, what if you need to add another test here outside of `doTest()`?

Reply:
I intend to add testing functions for another PR, I accidentally kept the data store testing class in this PR